lms/envs/devstack_docker.py

```diff
@@ -86,3 +86,22 @@
 
 if FEATURES['ENABLE_ENTERPRISE_INTEGRATION']:
     SYSTEM_WIDE_ROLE_CLASSES.extend(['enterprise.SystemWideEnterpriseUserRoleAssignment'])
+
+########################## THEMING  #######################
+# If you want to enable theming in devstack, uncomment this section and add any relevant
+# theme directories to COMPREHENSIVE_THEME_DIRS
+
+# We have to import the private method here because production.py calls
+# derive_settings('lms.envs.production') which runs _make_mako_template_dirs with
+# the settings from production, which doesn't include these theming settings. Thus,
+# the templating engine is unable to find the themed templates because they don't exist
+# in it's path. Re-calling derive_settings doesn't work because the settings was already
+# changed from a function to a list, and it can't be derived again.
+
+# from .common import _make_mako_template_dirs
```

Comment:
I don't think you need to explicitly import this, it should have brought in via a chain of imports by `from devstack import *` at the top of this file.

Reply:
It had to be explicitly imported because apparently `from X import *` doesn't import functions that start with an underscore. TIL!